### ISSUE TITLE
Staging Review finding: On mobile or at high zoom, the "Filter and sort" link doesn't lead anywhere

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#93690]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Transition Experience
**Product:** Transition Experience MVP v1
**Feature:** V1 Transition Experience

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When a user is on a mobile device or on a desktop at 300% zoom or higher, the filter and sort options are hidden. In their place is a "Filter and sort" link. However, clicking that link doesn't lead to any filter/sort mechanism, instead sending users to the top of the page.

<details><summary>Link, screenshot or steps to recreate</summary>

The "Filter and sort" link is coded as:
```
<va-link-action href="#" message-aria-describedby="Filter and sort" text="Filter and sort" type="secondary" class="hydrated"></va-link-action>
```

![Portion of page at high zoom with heading "Benefits to explore," an action link with text "Filter and sort," and a bold line of text describing the results currently on the screen.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/82863-Brian-1.png?raw=true)

</details>

## Recommended action

Links need to be coded with actual destinations, not just a placeholder `#`. Likewiise, functionality that's present on desktop must also be present on mobile.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Browser zoom
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by nkontrabecki** [comment]
Confirmed fixed and tested in Sprint 14.
